src/components/PropertyAmenities.jsx

```diff
@@ -67,11 +82,73 @@ class PropertyAmenities extends React.Component {
           }
         </styles.columns>
 
-        <styles.showMore>Show more</styles.showMore>
+        <styles.showMore onClick={this.handleSeeMore}>Show more</styles.showMore>
+
         {this.state.modal ?
```

Comment:
To clarify: if true, render the next html lines inside the } and if not skip over it?

Reply:
Exactly--But, I should use the logical && operator more because it seems cleaner for conditional rendering. If I have time, I'd like to refactor.